### PR TITLE
[#17541] fixed, added ext + functions

### PR DESCRIPTION
**Pull request by pop1989bb**
Fixed issue #17541: Fixed by adding the missing environment variable EXT, adding features to ConsoleApplication.php and the required asset directory.
Dev: pop1989_bb

### CODE REVIEW

File: application/core/ConsoleApplication.php [hunk 1]
```diff
@@ -149,4 +149,14 @@ public function getPluginManager()
         return $this->getComponent('pluginManager');
     }
 
+    public function getAssetManager()
```
Comment (olleharstedt) [inline application/core/ConsoleApplication.php:152]: Not sure these functions are needed, you should always be able to do: Yii::app()->assetManager, etc for other components.

Comment (pop1989bb) [reply application/core/ConsoleApplication.php:152]: We are using the ExportSurveyResultsService class from Limesurvey. This class is calling some internal functions which requires the added methods.

Comment (olleharstedt) [reply application/core/ConsoleApplication.php:152]: Hm, alright. Can you then please add docblocks? Like:

```
/**
 * @return CComponent
 */
 ```

Comment (Shnoulle) [reply application/core/ConsoleApplication.php:152]: asset manager and script are for web construction, then surely not loaded in console.

https://github.com/LimeSurvey/LimeSurvey/blob/3f1f5f17c3d7dd0aa7885d1bcbad583772ae3ccf/framework/web/CWebApplication.php#L149
Nothink in https://github.com/LimeSurvey/LimeSurvey/blob/master/framework/console/CConsoleApplication.php

**Pull request merged**